tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -72,9 +72,9 @@ def visit_functiondef(self, node):
         """
         try:
             if (
-                node.name == "__init__"
-                and node.parent.name.endswith("Client")
-                and node.parent.name not in self.ignore_clients
+                    node.name == "__init__"
+                    and node.parent.name.endswith("Client")
```

Comment:
Also just curious are all of the syntax edits from running black?

Reply:
Im not actually sure where those small syntax edits have come from

Reply:
unwanted spacing has been reverted back to the original